lib/fastlane/plugin/wpmreleasetoolkit/versioning/files/android_version_file.rb

```diff
@@ -0,0 +1,88 @@
+# A class for reading and writing Android version information to a version.properties file.
+require 'java-properties'
+
+module Fastlane
+  module Wpmreleasetoolkit
+    module Versioning
+      class AndroidVersionFile
+        attr_reader :version_properties_path
+
+        # Initializes a new instance of AndroidVersionFile with the specified version.properties file path.
+        #
+        # @param [String] version_properties_path The path to the version.properties file.
+        #
+        def initialize(version_properties_path:)
```

Comment:
Good point. I had left out a default value in case `version.properties` was stored somewhere else for whatever reason. But I agree that it still makes sense to have this as the default since the majority of RT clients will have it in the root directory.